roles/openshift_hosted_templates/files/v3.6/origin/registry-console.yaml

```diff
@@ -100,9 +100,9 @@ objects:
     redirectURIs:
       - "${COCKPIT_KUBE_URL}"
 parameters:
-  - description: "Container image name"
-    name: IMAGE_NAME
-    value: "cockpit/kubernetes"
+  - description: 'Specify "registry/namespace" prefix for container image; e.g. for "registry.example.com/cockpit/kubernetes:latest", set prefix "registry.example.com/cockpit/"'
+    name: IMAGE_PREFIX
```

Comment:
We should continue to support IMAGE_NAME.  We should add IMAGE_PREFIX as default('') so users can override if they need to specify a specific registry.

Reply:
See the invoking code; nothing ever actually set `IMAGE_NAME`, unless someone manually used the template. Essentially the template default was the hardcoded name.